### DISCUSSION TITLE
Is there another LTS release planned or selected beyond `1.14.x`

### DISCUSSION THREAD

**Discussion by cbgbt** [tokio-rs/tokio#4732] [Q&A]
Hello. We use tokio pretty extensively in a few projects. We're currently pinned to the older `1.8.x` LTS of tokio and were looking into upgrading, but noticed that the `1.14.x` LTS ends this month (6/2022). Are there plans for another LTS release of tokio? I'm wondering if we should wait for such a release, or reconsider our strategy for consuming new versions of tokio.

Sorry if this is answered elsewhere, I couldn't find references to it. Thanks!

**Comment by Darksonn** [comment]
Yeah, we will add a new one. I just opened #4733 to do so.

**Reply by cbgbt** [reply]
Thanks so much, that would be excellent!
